src/Database/Schema/MysqlSchema.php

```diff
@@ -25,9 +25,22 @@ class MysqlSchema extends BaseSchema
     /**
      * {@inheritDoc}
      */
-    public function listTablesSql($config)
+    public function listTablesSql($config, $type = 'both')
     {
-        return ['SHOW TABLES FROM ' . $this->_driver->quoteIdentifier($config['database']), []];
+        $table_type = null;
+        $sql = 'SHOW FULL TABLES FROM ' . $this->_driver->quoteIdentifier($config['database']);
+
+        if ($type != 'both') {
```

Comment:
Can we have constants for these string values? Constants allow for typo free method calls.